portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -13,64 +13,72 @@
 
 package com.google.sps.servlets;
 
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
 import com.google.appengine.api.datastore.DatastoreService;
 import com.google.appengine.api.datastore.DatastoreServiceFactory;
 import com.google.appengine.api.datastore.Entity;
 import com.google.gson.Gson;
 import java.io.IOException;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.*;
```

Comment:
Better to import what you want specifically from java.util.* rather than the whole package (like what you're doing above with ArrayList and List. It'll be faster since we won't be importing the whole thing. Are you using anything else form this package?

Reply:
Ah yes earlier I was testing if I needed to add another import and then forgot to take it out, whoops!